Nikschavan/header-footer-elementor
# Fix: Navigation Menu - Spacing getting added when if the menu has submenu in the footer.

### Description
Spacing getting added if the menu has submenu in the footer.

### Types of changes
Bugfix (non-breaking change which fixes an issue) 

### How has this been tested?
- Issue - https://share.getcloudapp.com/QwuKNj1r
- Create the menu with and without submenu.
- Use in the footer.
- Submenu should not get added in the footer if the menu has a submenu.
- Use the navigation menu in the header section too.
- Should not break existing functionality.
- Check the space on responsive devices too.
- Check with all layouts.
- https://trello.com/c/Ygze66vT/78-hfe-hamburger-nav-menu-adding-extra-space-at-the-bottom

### Checklist:
